internal/provider/provider.go

```diff
@@ -43,14 +49,51 @@ func (p *OnePasswordProvider) Metadata(ctx context.Context, req provider.Metadat
 func (p *OnePasswordProvider) Schema(ctx context.Context, req provider.SchemaRequest, resp *provider.SchemaResponse) {
 	resp.Schema = schema.Schema{
 		Attributes: map[string]schema.Attribute{
-			"url": schema.StringAttribute{
+			"connect_url": schema.StringAttribute{
 				MarkdownDescription: "The HTTP(S) URL where your 1Password Connect server can be found. Can also be sourced `OP_CONNECT_HOST` environment variable. Provider will use 1Password Connect server if set.",
 				Optional:            true,
+				Validators: []validator.String{
+					stringvalidator.ConflictsWith(
+						path.Expressions{
+							path.MatchRoot("url"),
+						}...,
+					),
+				},
 			},
-			"token": schema.StringAttribute{
+			"connect_token": schema.StringAttribute{
 				MarkdownDescription: "A valid token for your 1Password Connect server. Can also be sourced from `OP_CONNECT_TOKEN` environment variable. Provider will use 1Password Connect server if set.",
 				Optional:            true,
 				Sensitive:           true,
+				Validators: []validator.String{
+					stringvalidator.ConflictsWith(
+						path.Expressions{
+							path.MatchRoot("token"),
+						}...,
+					),
+				},
+			},
+			"url": schema.StringAttribute{
+				MarkdownDescription: "The HTTP(S) URL where your 1Password Connect server can be found. Can also be sourced `OP_CONNECT_HOST` environment variable. Provider will use 1Password Connect server if set. Deprecated: Use `connect_url` instead.",
+				Optional:            true,
+				Validators: []validator.String{
+					stringvalidator.ConflictsWith(
+						path.Expressions{
+							path.MatchRoot("connect_url"),
+						}...,
+					),
+				},
+			},
+			"token": schema.StringAttribute{
+				MarkdownDescription: "A valid token for your 1Password Connect server. Can also be sourced from `OP_CONNECT_TOKEN` environment variable. Provider will use 1Password Connect server if set. Deprecated: Use `connect_token` instead.",
+				Optional:            true,
+				Sensitive:           true,
```

Comment:
Depreciation message can be set as a property
```
DeprecationMessage:  "depriocation message for token",
```
So, `resp.Diagnostics.AddWarning("Deprecated field", "The \"url\" field is deprecated and will be removed in a future version. Use \"connect_url\" instead.")` and `resp.Diagnostics.AddWarning("Deprecated field", "The \"token\" field is deprecated and will be removed in a future version. Use \"connect_token\" instead.")` which are added manually, can be removed.

It will print warning like that:

<img width="497" height="200" alt="Image" src="https://github.com/user-attachments/assets/bcbbc3ae-55fa-418a-895c-4bfe7d6c5273" />